source/external-services/nextcloud/collabora.rst

```diff
@@ -0,0 +1,161 @@
+===========================
+Das Online-Office Collabora
+===========================
+
+.. sectionauthor:: `@rettich <https://ask.linuxmuster.net/u/rettich>`_
+
+Collabora Online ist eine angepasste Version von LibreOffice Online, einem Online-Office, welches sich auf dem Dockerhost betreiben lässt. 
+
+Mit Collabora können beispielsweise in Moodle und Nextcloud gleichzeitig mehrer Benutzer an einem Dokument arbeiten. Mit Collabora hat man so auch auf Tabletts oder Handys ein Office-Paket zur Verfügung. 
+
+Um Collabora auf dem Dockerhost zu installieren sind die gleichen Schritte wie bei der Nextcloud-Installation notwendig. 
+
+#. Erstellen eines Let's Encrypt - Zertifikats.
+#. Erstellen einer Site für die Collabora in nginx.
+#. Erstellen und Starten der Collabora App.
+
+.. hint:: Im Folgenden musst du natürlich ``office.meine-schule.de`` durch deine URL ersetzen.
+
+Erstellung des Zertifikats
+==========================
+
+Zuerst musst du dir Dienstenamen ausdenken und SSL-Zertifikat besorgen. Also z.B. office.meine-schule.de. 
+
+Dazu legst du einen DNS Eintrag für deine Dockerapp, z.B. office.meine-schule.de, der auf die IP des Dockerhosts zeigt an. Das darf auch ein CNAME sein.
+
+Trage diesen Host in die Datei ``/etc/dehydrated/domains.txt`` ein.
+
+Führe den Befehl ``dehydrated -c`` aus. Jetzt hast du die Zertifikate im Verzeichnis ``/var/lib/dehydrated/certs/`` zur Verfügung, der Docker Host aktualisiert diese per Cronjpb.
+
+Erstellen einer Site für Collabora in nginx
+===========================================
+
+Melde dich als root auf deinem Dockerhost an.
+
+Erstelle mit ``mkdir -p /srv/docker/office`` das Verzeichnis, in das alle Collabora-Dateien abgelegt werden.
+
+Erzeuge die Datei office.nginx.conf.
+
+.. code-block:: console
```

Comment:
..code-blocks:: console should not be used in this case, since the code does not refer to "Bash shell sessions". But simply code text that is not a programming language or anything else. Just use ..code::